### PR TITLE
Ansible pipelines deployed on wh-chaperone

### PR DESCRIPTION
**Pull request by scimerman**
- version updated from 4.5.1 to 4.5.5, as the version 4.5.1 had issues on wh-chaperone having missing `easybuild.main` file after installation (bug)
- the `PYTHONPATH` was incorrectly defined to
`/apps/software//EasyBuild/4.5.5/lib64/python3.6/site-packages/pip`
instead of
`/apps/software/EasyBuild/4.5.5/lib/python3.6/site-packages`

### CODE REVIEW

File: roles/install_easybuild/tasks/install_easybuild_with_pip.yml [hunk 1]
```diff
@@ -26,6 +26,13 @@
          "{{ pip_path.stdout }}" -V | grep -o 'lib[^ ]*python[^ ]*'
   register: python_install_suffix
   changed_when: true
+- name: 'Determine site-packages location for Lua module file to be correctly configured'
+  ansible.builtin.shell:
+    cmd: |
+         set -o pipefail
+         /usr/bin/find "{{ easybuild_software_dir }}/EasyBuild/{{ easybuild_version }}" -name "site-packages" | grep -o 'lib[^ ]*python[^ ]*'
+  register: pythonpath
+  changed_when: true
```
Comment (pneerincx) [inline roles/install_easybuild/tasks/install_easybuild_with_pip.yml:35]: If I understand correctly this replaces the `'Get install suffix from pip -V'` task, which can be deleted, right?

File: roles/install_easybuild/tasks/install_easybuild_with_pip.yml [hunk 2]
```diff
@@ -26,6 +26,13 @@
          "{{ pip_path.stdout }}" -V | grep -o 'lib[^ ]*python[^ ]*'
   register: python_install_suffix
   changed_when: true
+- name: 'Determine site-packages location for Lua module file to be correctly configured'
```
Comment (pneerincx) [inline roles/install_easybuild/tasks/install_easybuild_with_pip.yml:29]:
```suggestion
- name: 'Determine site-packages location for Lua module file.'
```

**Review by pneerincx — CHANGES_REQUESTED** [review]
See inline comments.

File: roles/install_easybuild/tasks/install_easybuild_with_pip.yml (hunk 1 above)
Comment (scimerman) [reply roles/install_easybuild/tasks/install_easybuild_with_pip.yml:35]: Good catch, but no.
This determine the PYTHONPATH that is works with `lib` and `lib64` pathing and similar change in structure (happened on one of the test system). 
The `pip` executable can be (and **is** on chaperone when you deploy `ansible-pipelines` from user's home folder) pointing to the user's virtual path.
Note that the `pip_path` variable is used only during the installation, and **not** anymore after that. When user runs the code, the executables are used from system and `PYTHONPATH` only.
What was already in current implementation is still valid. New block of code merely adds the script that detects the correct `PYTHONPATH` - > where all the packages ended up being installed.

Comment (pneerincx) [reply roles/install_easybuild/tasks/install_easybuild_with_pip.yml]: I still do not get it. The only thing the `Get install suffix from pip -V` task did was registering the `python_install_suffix` Ansible variable, which was only used to compile PYTHONPATH. This `python_install_suffix` variable is no longer used anywhere with this commit and PYTHONPATH is more reliably determined using the new `Determine site-packages location for Lua module file to be correctly configured` task. So why do we need to keep the `Get install suffix from pip -V` task?

Comment (scimerman) [reply roles/install_easybuild/tasks/install_easybuild_with_pip.yml]: You are correct. I checked again, and variable that was used in the templates is not being needed anymore. I removed the part with obsolete code.

**Review by pneerincx — APPROVED** [review]

**Pull request merged**